packages/admin/cms-admin/src/dam/DataGrid/fileUpload/useDamFileUpload.tsx

```diff
@@ -442,7 +442,7 @@ export const useDamFileUpload = (options: UploadDamFileOptions): FileUploadApi =
                         } else {
                             addValidationError(file, <UnknownError />);
                         }
-                    } else if (typedErr.response?.data.message === "Rejected File Upload: SVG must not contain JavaScript") {
+                    } else if (typedErr.response?.data.message.includes("SVG contains forbidden content")) {
```

Comment:
Out of scope: Checking for the error messages here is odd. Maybe we should introduce error codes?

Reply:
Yes, that would be better. I stuck with my old method of doing it here. But since we might rewrite the DAM this year, we can do it then.